src/cli/libs/utils.py

```diff
@@ -267,30 +271,59 @@ def deploy_servers(machines, token, ip_addresses):
     if len(machines) < 1:
         return
 
+    def get_node_labels(tags):
+        label_strings = []
+        for tag in tags:
+            tag_name = tag.name
+            if tag_name.startswith("LABEL_"):
+                _, label_name, label_value = tag_name.split("_", 3)
```

Comment:
Oops, sorry, wrong split here

```suggestion
                _, label_name, label_value = tag_name.split("_", 2)
```